rfcs/20221124-monorepo.md

```diff
@@ -248,7 +280,29 @@ To clean up the workflows we will define three new actions. The code for all thr
 
 #### Update workflows
 
-With this done, the development on the API and frontend can continue inside their subdirectories. The development of both parts will be independent. At least until we reach [long-term consolidation](#step-5-long-term-consolidation).
+Workflows with major changes:
+
+- `ci_cd.yml` from the API will absorb `ci.yml` from the frontend
```

Comment:
What approach will we take to avoid running API tests when only frontend code is changed and vice-versa? Combining the CI workflows will preclude us from using the workflow `files` filter. Did you have a different solution in mind?

Reply:
We can use the [`path-filter` action](https://github.com/dorny/paths-filter) to do that.